docs/topics/mta-cli-analyze-unsupported-provider.adoc

```diff
@@ -0,0 +1,24 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-07-30
+:_mod-docs-content-type: PROCEDURE
+
+[id="mta-cli-analyze-unsupported-provider_{context}"]
+= Analyzing a multi-language application for an unsupported language provider
+
+When analyzing a multi-language application with {ProductFullName} {CLINameTitle}, you can run the analysis for an unsupported language provider. To do so, you must override an existing supported language provider with your own unsupported language provider by using the `--provider-override` option. 
```

Comment:
This option is called `--override-provider-settings`

Reply:
Added a suggestion below